### ISSUE TITLE
Update server pings

### ISSUE THREAD

**Issue by bbondy** [brave/brave-browser#284]
the platform identifiers are:
`osx-bc`, `winx64-bc`, `winia32-bc` and `linux-bc`

Here are the params to use:
https://github.com/brave/browser-laptop/blob/master/app/updater.js#L131

Use prefs to store last time daily was true, last time monthly was true, etc. 
Basically follow the same logic as in updater.js.

To do the HTTPS request, you should use `SimpleURLLoader`.

This file has some important info too:
https://github.com/brave/browser-laptop/blob/master/js/constants/appConfig.js

In particular:
`updateHost` and the updates section here:
https://github.com/brave/browser-laptop/blob/master/js/constants/appConfig.js#L112

So basically just do a url request every `appUpdateCheckFrequency` and make sure to include those client side calculated params from prefs.
and time for tests, so maybe do `_unittest.cc` file to test some functions that do the logic.
